src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api/api';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [arePeopleLoading, setArePeopleLoading] = useState(false);
+  const [isLoadingSuccessful, setIsLoadingSuccessful] = useState(false);
+
+  const errorMessage = (
+    <p data-cy="peopleLoadingError" className="has-text-danger">
+      Something went wrong
+    </p>
+  );
+  const emptyPeopleMessage = (
+    <p data-cy="noPeopleMessage">There are no people on the server</p>
+  );
+
+  useEffect(() => {
+    setArePeopleLoading(true);
+    getPeople()
+      .then(peopleFromServer => {
+        setPeople(peopleFromServer);
+        setArePeopleLoading(false);
```

Comment:
You can put setArePeopleLoading(false); in `finaly` instead of using twice in `then` and `catch`